common/config/azure-pipelines/templates/core-build.yaml

```diff
@@ -6,7 +6,7 @@ parameters:
     default: $(System.DefaultWorkingDirectory)
   - name: rushBuildCacheEnabled
     type: number
-    default: 0
+    default: 1
   - name: rushBuildCacheWriteAllowed
```

Comment:
do we still need the rushBuildCacheWriteAllowed var? looks like its not used anymore, and we achieve the same thing with the shouldWriteToCache var

Reply:
We do not. Part of me liked 'hosting' it so it was visible, but unnecessary

Reply:
Nope - I missed that